examples/player/simid_player.html

```diff
@@ -116,7 +116,17 @@
     }
 
     function switchCreative(value) {
-      document.getElementById('creative_url').value = '../creatives/' + value;
+      const creative_url = document.getElementById('creative_url').value = '../creatives/' + value;
+
+      if (creative_url.includes("nonlinear")){
```

Comment:
this seems like its prone to a lot of things - one of which is the need to have the word 'nonlinear' in every nonlinear ad created.

Reply:
correct- this is just for our example player so I don't think it should be an issue. If you have any other suggestions let me know!

Reply:
You wont be needing this piece of code in your PR anyway. I’ll merge my autopopulate PR which has this functionality so you just have to pull from master before you merge

Reply:
yup, I commented on your PR

Reply:
Yeah, prefer just manually setting each one, to a trick like this (though I do like the idea).